website/components/Navbar/index.js

```diff
@@ -0,0 +1,54 @@
+import NavSection from 'components/NavSection';
+import StyledLink from 'components/StyledLink';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+import { SubNav, SubNavSections } from '@strapi/design-system/SubNav';
+import { Box } from '@strapi/design-system/Box';
+import { Icon } from '@strapi/design-system/Icon';
+import Discourse from '@strapi/icons/Discourse';
+import Github from '@strapi/icons/Github';
+import Image from 'next/image';
+import styled from 'styled-components';
+import { useTheme } from '@strapi/design-system/ThemeProvider';
+
+const HorizontalLine = styled.div`
```

Comment:
You can override a Box instead of use its props for background width and height :) (you'll only have to override the margin rule)